AdyenCardScanner/Sources/CardScannerViewController.swift

```diff
@@ -46,17 +46,17 @@ class CardScannerViewController: UIViewController {
         addOverlayView()
         observeRoiLayoutChanges()
 
-        viewModel.configureSession()
+        viewModel.viewDidLoad()
```

Comment:
These are lifecycle methods of a `UIViewController`, what is the reason to leak them to the view model layer? This approach might be fragile if we ever decide using this outside of UIKit (think of SwiftUI for example).